docs/guides/modules/deploy/pages/set-up-rollbacks.adoc

```diff
@@ -35,97 +35,88 @@ a| * Simple systems
+. Select btn:[Commit config & create Pull Request] to create this rollback configuration template in your repository. The setup will open a pull request in a new tab with the suggested changes.
++
+NOTE: The configuration will be committed to a new branch called `rollback-pipeline-setup` in the repository you selected .
++
+image::guides:ROOT:deploy/commit-config-button.png[Commit config button]
 
-==== Configuration tips
+. Alternatively, you can copy the suggested configuration into an existing or newly created rollback config file. Make sure the file is called `rollback.yml` and is located in the `.circleci` folder, for the system to detect and use this file. In this case,select btn:[I already have a rollback config] and skip to the next step. See the <<configuration-tips, Configuration tips>> section for more details on customizing your rollback configuration.
++
+image::guides:ROOT:deploy/already-have-config-button.png[Already have config button]
 
-When customizing your rollback configuration, you can use the following pipeline values to access rollback values:
+. You can manually navigate to the pull request by selecting btn:[View pull Request].
++
+image::guides:ROOT:deploy/view-pr-button.png[View pull request button]
 
-* `pipeline.deploy.component_name`
-* `pipeline.deploy.environment_name`
-* `pipeline.deploy.target_version`
-* `pipeline.deploy.current_version`
-* `pipeline.deploy.namespace`
-* `pipeline.deploy.reason`
+. In the pull request in your repository, modify the rollback configuration according to your specific deployment needs. For example: set your own parameters and rollback logic, uncomment the included common rollback setups or write your own custom rollback implementation.
+. Once you have customized the configuration, merge the pull request to complete the rollback setup.
 
-==== Deploy markers for rollbacks
-You can use deploy markers with the `--rollback` flag to indicate rollback deployment:
 
-[source,bash]
-----
-circleci run release plan \
-          --environment-name=${ENVIRONMENT_NAME} \
-          --namespace=${NAMESPACE} \
-          --component-name=${COMPONENT_NAME} \
-          --target-version=${TARGET_VERSION} \
-          --rollback
-----
+=== Complete rollback setup
 
-You can also update the status of the rollback deployment as mentioned in the xref:configure-deploy-markers.adoc[Configure Deploy Markers] guide to reflect the state of the rollback accurately in the CircleCI UI.
+. After merging the pull request, return to the setup page in the CircleCI web app.
+. Select btn:[Setup rollback pipeline] to activate the rollback pipeline.
++
+image::guides:ROOT:deploy/setup-rollback-pipeline-button.png[Setup rollback pipeline button]
 
-=== 6. Access rollback functionality
-Once the pull request is merged, the rollback setup is complete. You can now use the rollback functionality in the CircleCI UI.
+You can now use the rollback functionality in the CircleCI UI.
```

Comment:
You can now rolback deployments by running a rollback pipeline

or something like that

because they can already use a workflow rollback without all this work